packages/common/src/chains.ts

```diff
@@ -31,3 +37,29 @@ export const pgnTestnet: Chain = {
   },
   testnet: true,
 };
+
+export const pgn: Chain = {
+  id: 424,
```

Comment:
Could we use `ChainId.PGN` here, so that numeric id's are only declared in the ChainId enum?

(Ah, but I see that the `ChainId` enum wasn't introduced in this PR, just moved around.)

Reply:
We could, but this custom definition is going to be upstreamed to wagmi anyway, so I would leave it.